modules/custom/az_person/src/Plugin/Block/AZPersonReorderNotice.php

```diff
@@ -23,9 +24,41 @@ class AZPersonReorderNotice extends BlockBase {
    */
   public function build() {
 
-    return [
-      '#markup' => '<div class="callout callout-leaf"><p><strong>Need to reorder people?</strong></p><p>Use the button below to reorder the people on this page.</p><p><a class="btn btn-success" href="/admin/az-person/reorder-people">Reorder People</a></p></div>',
+    $callout_markup_header = [
+      '#markup' => '<p><strong>' . $this->t('Need to reorder people?') . '</strong></p>',
     ];
+    $callout_markup_description = [
+      '#markup' => '<p>' . t('Use the button below to reorder the people on this page.') . '</p>',
```

Comment:
```suggestion
      '#markup' => '<p>' . $this->t('Use the button below to reorder the people on this page.') . '</p>',
```

Reply:
FWIW, I'd be in favor of replacing the blocks with contextual links, if possible.